src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,17 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.12.1
+
+### Bug fixes
```

Comment:
we can follow the same format we already used in previous point releases : https://submariner.io/community/releases/#v0111